code/game/turfs/change_turf.dm

```diff
@@ -164,10 +164,19 @@ GLOBAL_LIST_INIT(blacklisted_automated_baseturfs, typecacheof(list(
 			QDEL_NULL(stashed_air)
 			return
 		var/turf/open/newTurf = .
+		var/turf_fire_ref
+		if(turf_fire)
+			if(ispath(path, /turf/open/openspace) || ispath(path, /turf/open/space))
```

Comment:
again, you dont need to operate on paths. you already created the turf.

Reply:
Agony, suffering even